spec/order_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'order'
+require 'menu'
+require 'bill'
+
+describe Order do
+  describe "#add_to_order" do
+    it "Should add items to a user's order" do
+      expect { subject.add_to_order("Egg Fried Rice", 2) }.to change { subject.order.length }.by(1)
```

Comment:
Maybe consider testing output directly again, as you're not instantiating an object just creating an order hash, it would useful to also confirm that you `"Egg Fried Rice", 2` really was added and is a hash as this test could still pass by just adding an empty object to `subject.order`.